control/velocity_controller/package.xml

```diff
@@ -1,23 +1,29 @@
 <?xml version="1.0"?>
-<package format="2">
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
   <name>velocity_controller</name>
-  <version>0.1.0</version>
-  <description>
-    This package provides a vehicle velocity controller.
-  </description>
-
+  <version>0.0.1</version>
+  <description>vehicle velocity controller as a ROS 2 node</description>
   <maintainer email="makoto.kurihara@tier4.jp">Makoto Kurihara</maintainer>
+  <maintainer email="horibe.takamasa@gmail.com">Takamasa HORIBE</maintainer>
+  <license>Apache License 2.0</license>
 
-  <license>Appache2</license>
+  <buildtool_depend>ament_cmake</buildtool_depend>
+  <build_depend>rclcpp</build_depend>
```

Comment:
I think you should be a `<depend>`, else you miss one of the three depend properties for another package depending on this one

Reply:
Thanks. Changed to <depend>.